src/components/filter/MessagesFilterPanel.tsx

```diff
@@ -104,9 +106,16 @@ const MessagesFilterPanel = () => {
 		);
 	}, [filter, filterStore.filter, streams, isSoftFilterApplied]);
 
-	const isLoading = computed(
+	const stopLoading = React.useCallback(() => {
+		messagesDataStore.stopMessagesLoading();
+		workspaceStore.stopAttachedMessagesLoading();
+	}, []);
+
+	const isAnchorLoading = computed(() => workspaceStore.isLoadingAttachedMessages).get();
```

Comment:
can we rename it to `isLoadingAttachedMessages`? or do we even need isAnchorLoading computed value ? can we just pass it to the component that's using it